RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate AddExistingChannelView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Create a Team, then click on "edit/create" icon on the header, and the new header title should be Add Channel to Team, so you need to click on `Add Existing`
- Then check if is working properly the query, search, and other functions.

## Screenshots

## Types of changes




## Checklist



## Further comments
